sdk/src/metrics/meter.cc

```diff
@@ -149,7 +179,30 @@ const sdk::instrumentationlibrary::InstrumentationLibrary *Meter::GetInstrumenta
 
 MeasurementProcessor *Meter::GetMeasurementProcessor() const noexcept
 {
-  return context_->GetMeasurementProcessor();
+  return meter_context_->GetMeasurementProcessor();
+}
+
+std::unique_ptr<WritableMetricStorage> Meter::RegisterMetricStorage(
+    InstrumentDescriptor &instrument_descriptor)
+{
+
+  auto view_registry = meter_context_->GetViewRegistry();
+  std::unique_ptr<WritableMetricStorage> storages(new MultiMetricStorage());
+
+  auto result = view_registry->FindViews(
```

Comment:
What `storages` will be if we didn't find `view` here?
also `result` is not used.

Reply:
[ViewRegistry::FindViews](https://github.com/open-telemetry/opentelemetry-cpp/blob/9157fd474883b4d90dab2e4f5aa887a705c2ab94/sdk/include/opentelemetry/sdk/metrics/view/view_registry.h#L65) returns a default view (local static) of drop aggregation type if no view is found.

Reply:
Thanks for the clarification.